rubygem-concurrent-ruby-edge/rubygem-concurrent-ruby-edge.spec

```diff
@@ -0,0 +1,76 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name concurrent-ruby-edge
+
+Summary: Edge concepts for the modern concurrency tools for Ruby
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.1.0.pre3
+Release: 1%{?dist}
+Group: Development/Languages
+
+License: MIT
+URL: https://github.com/ruby-concurrency/concurrent-ruby
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+%if "%{?scl}" == "ruby193" || (0%{?rhel} == 6 && "%{?scl}" == "")
```

Comment:
Ah yes, apipie-bindings is built for non-SCL usage.  I'd tend to pick something new, like fog-libvirt or even dynflow!

Sorry for the confusion, I'll get back to my gem2spec templates soon..